### ISSUE TITLE
Admin Dashboard: "Disabling a user's Account" is not implemented.

### ISSUE THREAD

**Issue by CamiMonte** [slashroots/harvest-v2#161]

**Comment by CamiMonte** [comment]
The functionality to disable a user's account has not been implemented

**Comment by paradoxxjm** [comment]
@CamiMonte was an UI proposal created for this feature?

**Comment by CamiMonte** [comment]
@paradoxxjm Yes, a UI proposal was created for this feature. The following describes the interactions as designed.

This feature can be accessed via:

1. Clicking the "Disable" link in the "User Accounts" table.
_(See below Low-Fidelity Prototype demonstrating this interaction.)_

![dev dashboard - no apps](https://cloud.githubusercontent.com/assets/20387220/23176691/2f818930-f832-11e6-88cd-4c007f383da7.png)
![disable user - modal](https://cloud.githubusercontent.com/assets/20387220/23176713/480b4d6a-f832-11e6-85b1-8e120dad148c.png)


2. Changing the user's Status on the users' Account Profile screen. 
_(See below Low-Fidelity Prototype demonstrating this interaction.)_

![user accounts profile 2](https://cloud.githubusercontent.com/assets/20387220/23176830/b98cf3d0-f832-11e6-83a1-985399fdd906.png)
![user accounts - edit profile - app tab](https://cloud.githubusercontent.com/assets/20387220/23176971/36e526f4-f833-11e6-8ef3-019468041891.png)
